src/Tickets/Admin/Tickets/List_Table.php

```diff
@@ -497,12 +543,23 @@ public function column_start( $item ): string {
 		$date_format = tribe_get_date_format( true );
 		$ts          = $item->start_date();
 
-		return sprintf(
+		$start = sprintf(
 			'<time datetime="%1$s" title="%2$s">%3$s</time>',
 			esc_attr( \Tribe__Date_Utils::reformat( $ts, 'c' ) ),
 			esc_html( \Tribe__Date_Utils::reformat( $ts, get_option( 'date_format' ) . ' ' . get_option( 'time_format' ) ) ),
 			esc_html( \Tribe__Date_Utils::reformat( $ts, $date_format ) )
 		);
+
+		/**
+		 * Filters the start date for the All Tickets Table.
+		 *
+		 * @since TBD
+		 *
+		 * @param string $start The start date for the All Tickets Table.
+		 *
+		 * @return string
+		 */
+		return apply_filters( 'tec_tickets_all_tickets_table_column_start', $start, $item );
```

Comment:
```suggestion
		return apply_filters( 'tec_tickets_all_tickets_table_column_start_date', $start, $item );
```
Let's be clear in the filter that it's the start _date_